src/components/TableNew/TableHeaderCell.tsx

```diff
@@ -0,0 +1,19 @@
+import { cn } from '@/lib/utils'
+import { FC } from 'react'
+import { CommonComponentProps } from '../commonProps'
+
+export interface TableHeaderCellProps extends CommonComponentProps {
+  id: string // columnName
+  onClick: () => void
+}
+
+export const TableHeaderCell: FC<TableHeaderCellProps> = ({ children, className, onClick }) => {
+  return (
+    <div
+      className={cn('flex flex-row cursor-pointer justify-center items-center', className)}
+      onClick={onClick}
```

Comment:
I'm not completely sure if we should assume that all the header cells are clickable. I think we should allow them to do it, but I'd avoid this behaviour by default